sdk/messaging/azservicebus/internal/errors.go

```diff
@@ -91,17 +91,18 @@ var amqpConditionsToRecoveryKind = map[amqp.ErrorCondition]recoveryKind{
 	amqp.ErrorCondition("com.microsoft:operation-cancelled"): RecoveryKindNone,
 
 	// Link recovery needed
-	amqp.ErrorDetachForced: RecoveryKindLink, // "amqp:link:detach-forced"
+	amqp.ErrorDetachForced:          RecoveryKindLink, // "amqp:link:detach-forced"
+	amqp.ErrorTransferLimitExceeded: RecoveryKindLink,
```

Comment:
Doc comment

Reply:
Added. There were a few others, so I added comments for them as well.